.github/workflows/publish.yml

```diff
@@ -69,7 +65,7 @@ jobs:
       - name: releae to VSCode marketplace
         run: |
           if [ "${{ github.event.inputs.preview }}" == "true" ]; then
-            vsce publish --pre-release --azure-credential --packagePath ./release/*.vsix --noVerify
+            npx @vscode/vsce publish --pre-release --azure-credential --packagePath ./release/*.vsix --noVerify
```

Comment:
Just curious, what's the differnce with `npm install @vscode/vsce -g`?

Reply:
no different, this change just make sure that the intermediate steps do not download vsce instead of @vscode/vsce